lib/Conversion/TorchToMhlo/Basic.cpp

```diff
@@ -1292,6 +1292,51 @@ LogicalResult ConvertAtenOp<AtenArangeStartStepOp>::matchAndRewrite(
   return success();
 }
 
+// AtenMaskedFillScalarOp
+template <>
+LogicalResult ConvertAtenOp<AtenMaskedFillScalarOp>::matchAndRewrite(
```

Comment:
I think we can decompose AtenMaskedFillScalarOp into `AtenWhereScalarSelfOp`, which is more straightforward

Reply:
And the `AtenWhereScalarSelfOp` can be convert into `mhlo::BroadcastSelectOp`